xml/System.Composition.Hosting/CompositionHost.xml

```diff
@@ -66,10 +66,10 @@
         </Parameter>
       </Parameters>
       <Docs>
-        <param name="providers">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="providers">The providers, containing export descriptions for the parts being composed.</param>
+        <summary>Creates the composition host with the specified collection of providers.</summary>
```

Comment:
change this to "specified array of providers" to disambiguate it from the other overload.